vaadin-spring/src/main/java/com/vaadin/flow/spring/SpringVaadinServletService.java

```diff
@@ -22,6 +22,8 @@
 import java.util.Optional;
 import java.util.stream.Collectors;
 
+import org.springframework.beans.factory.NoSuchBeanDefinitionException;
+import org.springframework.boot.autoconfigure.web.WebProperties.Resources;
```

Comment:
Is this class to be expected to work with plain spring or just spring boot?

Reply:
It is supposed to work with Spring. Seems like there are no proper tests for plain Spring projects in this repository

Reply:
Let's keep the untested code then and hope that it might work with plain Spring..

Reply:
I would like to have a more spring boot oriented implementation as well.. but I've seen multiple PRs in this repo in the past where explicitly a more complex route was taken to reduce the coupling with spring boot.. that's why I wanted to mention it